samples/net/echo_client/prj_cc2520.conf

```diff
@@ -35,7 +34,6 @@ CONFIG_NET_STATISTICS_PERIODIC_OUTPUT=y
 
 CONFIG_NET_L2_IEEE802154=y
 CONFIG_NET_DEBUG_L2_IEEE802154=y
-CONFIG_NET_L2_IEEE802154_FRAGMENT=y
 CONFIG_NET_DEBUG_L2_IEEE802154_FRAGMENT=y
```

Comment:
ditto and for all other files below

Reply:
hum here fragment debugging is set to yes, where no is the default.